spec/integration/blueprint_with_ca_spec.rb

```diff
@@ -0,0 +1,60 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata'
+
+describe 'Create project using GoodData client', :constraint => 'slow' do
+  before(:all) do
+    @client = ConnectionHelper.create_default_connection
+    @blueprint = GoodData::Model::ProjectBlueprint.build("My project from blueprint") do |project_builder|
+      project_builder.add_date_dimension('created_on')
```

Comment:
In order to keep things simple, please create only metadata that we need for the test.